### PR TITLE
Change column type from String to LowCardinality

### PR DESCRIPTION
**Pull request by alexon1234**
Change column `browser` of `analytics_pages_mv.datasource` from String to LowCardinality(String)

Closes https://github.com/tinybirdco/use-case-examples/issues/39

### CODE REVIEW

File: change_column_type_to_lowcardinality/pipes/analytics_pages.pipe [hunk 1]
```diff
@@ -0,0 +1,28 @@
+VERSION 1
```
Comment (albertojuan) [inline change_column_type_to_lowcardinality/pipes/analytics_pages.pipe:1]: If I'm not wrong we're deprecating this, so shouldn't be used as example

File: utils/populate_with_backfill.sh [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    else
+        echo "Counts are not equal."
+        sleep $retry_interval
+        retry_count=$((retry_count + 1))
+    fi
+done
+
+if [[ $retry_count -eq $max_retries ]]; then
+    exit 1
+fi
```
Comment (albertojuan) [inline utils/populate_with_backfill.sh:59]: Awesome 👏!

Comment (elenatorro) [reply utils/populate_with_backfill.sh:59]: About this, wouldn't it make sense to reuse the scripts we have in the [CI repo](https://github.com/tinybirdco/ci/tree/main/scripts)?

File: change_column_type_to_lowcardinality/pipes/analytics_pages.pipe (hunk 1 above)
Comment (alexon1234) [reply change_column_type_to_lowcardinality/pipes/analytics_pages.pipe:1]: But are we using `v3` or `v2` in our examples?

**Review by xavijam — APPROVED** [review]
🚢 it!

File: utils/populate_with_backfill.sh (hunk 2 above)
Comment (xavijam) [reply utils/populate_with_backfill.sh]: Yep, I feel it is the same

Comment (alexon1234) [reply utils/populate_with_backfill.sh]: Sure, but to do this how we would do it? A git submodule?

File: change_column_type_to_lowcardinality/README.md [hunk 3]
```diff
@@ -1,3 +1,49 @@
...
+         toDate(timestamp) AS date,
+         device,
+-        browser,
++        toLowCardinality(browser) as browser,
+         location,
+         pathname,
+         uniqState(session_id) AS visits,
+```
+
+Then, we will need to increase the `VERSION` inside `.tinyenv` to indicate we are going to release a new version
```
Comment (xavijam) [inline change_column_type_to_lowcardinality/README.md:40]: With `tb release generate --semver 0.0.1` will do the trick with this and creating the CI and CD files.

Comment (alexon1234) [reply change_column_type_to_lowcardinality/README.md]: I've updated the README to indicate they can use that command for creating the scaffolding

**Pull request merged**